src/handler_greeter.erl

```diff
@@ -0,0 +1,21 @@
+
+-module(handler_greeter).
```

Comment:
I think the name of this file and module is better off being prefixed with "xqerl_". Reason being is the flat namespace of Erlang. There is a higher chance when someone is embedding xqerl into their own project that this name here could cause a conflict.

Reply:
OK.  everything should be renamed in latest push